### DISCUSSION TITLE
Gestion des tokens

### DISCUSSION THREAD

**Discussion by MaelTheuliere** [spyrales/gouvdown#48] [General]
Hello, 
@RLesur le token github_pat n'est plus valide semble t il 

https://github.com/spyrales/gouvdown/runs/5376047923?check_suite_focus=true#step:6:1

```
Run remotes::install_deps(dependencies = TRUE, type = "mac.binary")
Error: Error: HTTP error 401.
  Bad credentials

  Rate limit remaining: 59/60
  Rate limit reset at: 2022-03-01 14:40:19 UTC

  
Execution halted
Error: Process completed with exit code 1.
```
Je n'ai pas la main pour créer un token sur spyrales, tu peux me rajouter dans les admin ? 
ou le faire ;-)

**Comment by jdlom** [comment]
@RLesur @MaelTheuliere Est-ce que vous avez réglés ce problème ? Les droits ont été donnés ?

**Reply by MaelTheuliere** [reply]
pas à ma connaissance @RLesur ?
